projects/angular-grid-layout/src/lib/grid-item/grid-item.component.html

```diff
@@ -1,2 +1,9 @@
-<ng-content></ng-content>
-<div #resizeElem class="grid-item-resize-icon"></div>
+<div #container class="ktd-grid-item-container">
+    <ng-content></ng-content>
+    <div #resizeElem class="grid-item-resize-icon"></div>
+</div>
+
+<div class="ktd-grid-item-placeholder" #placeholderContainer>
+    <ng-container *ngTemplateOutlet="placeholder?.templateRef ?? default"></ng-container>
```

Comment:
Placeholder template should be created only when the user starts a drag. This way we prevent having N placeholders created in background. This is not an issue if placeholders are just colored divs, but it can be when they are entire components.
Sorry for not pointing that out on the feature definition, but with the placeholder inside the grid items, it should be that way. If you have doubts you can contact me about it.

Reply:
Continuing with this 'on demand' placeholder creation, it may be better if we create it at grid level, not necessarily on grid-item level. We need to pass it at grid-item level, but I think we should create it afterwards on grid level. This way we can keep grid-item clean also, no changes needed, only the projection of the ng-template.